packages/notifi-react-card/lib/components/AlertHistory/AlertNotificationRow.tsx

```diff
@@ -8,7 +8,7 @@ export type AlertNotificationViewProps = Readonly<{
   notificationImage?: JSX.Element;
   notificationSubject: string | undefined;
   notificationDate: string;
-  notificationMessage: string | undefined;
+  notificationMessage?: string | undefined;
```

Comment:
let's not make this an optional prop -- it would be better if consumers called it with an explicit `undefined` value